Resources/Prototypes/_Goobstation/Entities/Clothing/Eyes/glasses.yml

```
@@ -15,12 +15,17 @@
     tags:
     - WhitelistChameleon
     - HudMedical
+    - HudSecurity
   - type: IdentityBlocker
     coverage: EYES
   - type: ShowHealthBars
     damageContainers:
     - Biological
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 5

(indentation)